liquidity/liquidity.go

```diff
@@ -928,17 +935,13 @@ func (m *Manager) checkExistingAutoLoops(ctx context.Context,
 	return &summary, nil
 }
 
-// getEligibleChannels takes lists of our existing loop out and in swaps, and
-// gets a list of channels that are not currently being utilized for a swap.
-func (m *Manager) getEligibleChannels(ctx context.Context,
-	loopOut []*loopdb.LoopOut, loopIn []*loopdb.LoopIn) (
-	[]lndclient.ChannelInfo, error) {
+// ineligibleChannels examines our existing swaps and returns a summary of the
+// current activity which can be used to determine whether we should perform
+// any swaps.
+func (m *Manager) ineligibleChannels(loopOut []*loopdb.LoopOut,
```

Comment:
nit: imo it's a bit confusing to call this fn `ineligibleChannels`. How about simply calling it `currentSwapTraffic` since that's what it does?

Reply:
done!